pkg/daemon/update.go

```diff
@@ -427,6 +430,8 @@ func calculatePostConfigChangeActionFromFileDiffs(oldIgnConfig, newIgnConfig ign
 		"/var/lib/kubelet/config.json",
 	}
 	filesPostConfigChangeActionReloadCrio := []string{
+		GPGNoRebootPath,
```

Comment:
This question strike me a bit late, but do we need to reload crio for updating GPG key path?

Reply:
I'm not aware of any guarantee that the GPG key is re-read every time it is used as opposed to loaded from memory, so that's why I thought reload for both